python/lsst/ts/standardscripts/base_track_target.py

```diff
@@ -278,14 +299,20 @@ async def configure(self, config):
 
         if hasattr(self.config, "slew_icrs"):
             self.slew_type = SlewType.ICRS
+        elif hasattr(self.config, "slew_planet"):
+            self.slew_type = SlewType.PLANET
         elif hasattr(self.config, "find_target"):
             self.slew_type = SlewType.AZEL
 
         self.log.debug(f"Slew type: {self.slew_type!r}.")
 
         await self.configure_tcs()
 
-        self.config.rot_type = getattr(RotType, self.config.rot_type)
+        if self.slew_type is not SlewType.PLANET:
+            self.config.rot_type = getattr(RotType, self.config.rot_type)
+        else:
+            self.config.rot_type = RotType.Sky
```

Comment:
I think this has no effect if the slew type is `PLANET`. I think we have 2 option here either raise a `RuntimeError` exception if the user select `PLANET` with `rot_type != Sky` or log a warning message. I am really torn between both options, but I slightly lean towards just warning and giving it a pass..